packages/create-remix/package.json

```diff
@@ -17,31 +17,10 @@
     "create-remix": "dist/cli.js"
   },
   "dependencies": {
-    "arg": "^5.0.1",
-    "chalk": "^4.1.2",
-    "execa": "5.1.1",
-    "gunzip-maybe": "^1.4.2",
-    "log-update": "^5.0.1",
-    "node-fetch": "^2.6.9",
-    "proxy-agent": "^6.3.0",
-    "rimraf": "^4.1.2",
-    "semver": "^7.3.7",
-    "sisteransi": "^1.0.5",
-    "sort-package-json": "^1.55.0",
-    "strip-ansi": "^6.0.1",
-    "tar-fs": "^2.1.1"
-  },
-  "devDependencies": {
-    "@types/gunzip-maybe": "^1.4.0",
-    "@types/node-fetch": "^2.5.7",
-    "@types/tar-fs": "^2.0.1",
-    "fs-extra": "^10.0.0",
-    "msw": "^0.39.2",
-    "tiny-invariant": "^1.2.0",
-    "esbuild-register": "^3.3.2"
+    "@remix-run/dev": "1.19.1"
```

Comment:
Is replacing all of the dependencies intended? The build seems to be failing because of it.

Reply:
I must have messed up a rebase 🙈😅